ledger/src/blockstore.rs

```diff
@@ -1636,14 +1636,16 @@ impl Blockstore {
         })
     }
 
-    pub fn get_data_shreds_for_slot(
-        &self,
-        slot: Slot,
-        start_index: u64,
-    ) -> std::result::Result<Vec<Shred>, shred::Error> {
+    pub fn get_data_shreds_for_slot(&self, slot: Slot, start_index: u64) -> Result<Vec<Shred>> {
         self.slot_data_iterator(slot, start_index)
             .expect("blockstore couldn't fetch iterator")
-            .map(|data| Shred::new_from_serialized_shred(data.1.to_vec()))
+            .map(|data| {
```

Comment:
nit: preexisting code, but you could change this from `|data|` to `|(_, serialized_shred)|` and replace `data.1` with `serialized_shred`.

Reply:
I like it!